src/WireMock.Net.Aspire/WireMockInspector.cs

```diff
@@ -0,0 +1,43 @@
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Aspire.Hosting.WireMock;
+
+internal static class WireMockInspector
+{
+    /// <summary>
+    /// Opens the WireMockInspector tool to inspect the WireMock server.
+    /// </summary>
+    /// <param name="wireMockUrl"></param>
+    /// <param name="title"></param>
+    /// <exception cref="InvalidOperationException"></exception>
+    /// <remarks>
+    /// Copy of <see href="https://github.com/WireMock-Net/WireMockInspector/blob/main/src/WireMock.Net.Extensions.WireMockInspector/WireMockServerExtensions.cs" />
+    /// without requestFilters and no call to WaitForExit() method in the process so it doesn't block the caller.
+    /// </remarks>
+    public static void Inspect(string wireMockUrl, [CallerMemberName] string title = "")
+    {
+        try
+        {
+            var arguments = $"attach --adminUrl {wireMockUrl} --autoLoad --instanceName \"{title}\"";
+
+            Process.Start(new ProcessStartInfo

```

Comment:
Will this process be killed when Aspire stops?

Reply:
I just tested this, and yes, it is being killed when the AppHost is killed. I assume that is because the entire process tree is killed, but I'm not sure. Do we not want it to stop?